app/src/main/res/values-it/strings.xml

```diff
@@ -165,6 +165,9 @@
     <string name="settings_su_reauth_summary">Richiedi nuovamente i permessi di root dopo l\'aggiornamento di un\'app</string>
     <string name="settings_su_tapjack_title">Protezione anti-tapjacking</string>
     <string name="settings_su_tapjack_summary">La schermata di richiesta dei permessi di root non risponderà al tocco mentre è oscurata da altre finestre o elementi in sovraimpressione</string>
+    <string name="settings_su_auth_title">Autenticazione utente</string>
+    <string name="settings_su_auth_summary">Richiedi l\'autenticazione utente durante le richieste di Superutente</string>
```

Comment:
Thank you for the PR! I suggest changing this to:
```suggestion
    <string name="settings_su_auth_summary">Richiedi l\'autenticazione dell\'utente per le richieste di accesso root</string>
```
"Superuser" is consistently translated into "accesso root" throughout the application.